Dockerfile

```diff
@@ -0,0 +1,8 @@
+from node:8
+label maintainer="michael@brunton-spall.co.uk"
+label uk.co.brunton-spall.version="0.0.1-beta"
```

Comment:
what is this for?

Reply:
I believe this is the replacement for the old MAINTAINER deprecation

Reply:
the one above it is,
haven't seen anything like this line before,
also you can make it multiline to save layers in the image:
https://docs.docker.com/engine/reference/builder/#label